meson.build

```diff
@@ -0,0 +1,266 @@
+# meson build for open-isns
+
+project('libisns', 'c',
+  version: '0.101',
+  license: 'LGPL-2.1-or-later',
+  default_options: [
+    'c_std=gnu99',
+    'warning_level=2',
+    'buildtype=release',
+    'libdir=/usr/lib64',
+    'prefix=/usr']
+  )
+
+# get feature options
+with_security = get_option('security').enabled()
+with_slp = get_option('slp').enabled()
```

Comment:
You're casting these to a boolean, true if enabled, false if auto (commonly called "check" in autotools land) / disabled.

The main advantage of a feature option, though, is that you can pass the option object itself as e.g. `security_dep = dependency('libcrypto', required: get_option('security'))`

In the enabled case, which you currently handle, this is equivalent: the dependency is mandatory when enabled.

In the disabled case, which you also handle, this is equivalent: the dependency is not even looked up. The UX is a bit different: when Meson *knows* there's a disabled feature, it prints "Dependency **libcrypto** skipped: feature **security** disabled".

In the auto case, you treat it like disabled, but Meson would look it up, try to use it if it's installed, but continue if it's not installed. autotools equivalent: `--with-security=check`.

In all cases, you can determine whether the security feature is ultimately merged into the build definitions with `if security_dep.found()`, or even:

```
conf.set('WITH_SECURITY', security_dep.found())
```

Reply:
cool. Fixed.